addons/api/addon/models/storage-bucket.js

```diff
@@ -13,6 +13,14 @@ export const TYPES_STORAGE_BUCKET_PLUGIN = Object.freeze([
   TYPE_STORAGE_BUCKET_PLUGIN_AWS_S3,
 ]);
 
+export const TYPE_CREDENTIAL_STATIC = 'static';
+export const TYPE_CREDENTIAL_DYNAMIC = 'dynamic';
+
+export const TYPES_CREDENTIALS = Object.freeze([
```

Comment:
None blocking comment: What is the purpose of TYPES_CREDENTIALS being an `object.freeze`?

Reply:
This is the pattern we are following when creating new type constants to prevent modifications